tests/functional/SignatureModuleTest.php

```diff
@@ -236,7 +237,7 @@ protected function validate($path, $signatureFieldName, $certificate)
         while (\openssl_error_string());
 
         $pkey = \openssl_pkey_get_public($certificate);
-        $res = \openssl_verify($data, $signatureValue, $pkey, \SetaPDF_Signer_Digest::getOpenSslInt($digest));
+        $res = \openssl_verify($data, $signatureValue, $pkey, $digest);
```

Comment:
The whole `validate()` method can be replaced by:

        $document = \SetaPDF_Core_Document::loadByFilename($path);
        $result = \SetaPDF_Signer_ValidationRelatedInfo_IntegrityResult::create($document, $signatureFieldName);

        return $result->isValid();